packages/atomic/src/components/commerce/atomic-product-description/atomic-product-description.ts

```diff
@@ -0,0 +1,167 @@
+import {Schema, StringValue} from '@coveo/bueno';
+import type {Product} from '@coveo/headless/commerce';
+import {html, LitElement, nothing, unsafeCSS} from 'lit';
+import {customElement, property, state} from 'lit/decorators.js';
+import {createRef} from 'lit/directives/ref.js';
+import {booleanConverter} from '@/src/converters/boolean-converter.js';
+import {bindingGuard} from '@/src/decorators/binding-guard.js';
+import {bindings} from '@/src/decorators/bindings.js';
+import {createProductContextController} from '@/src/decorators/commerce/product-template-decorators.js';
+import {errorGuard} from '@/src/decorators/error-guard.js';
+import {injectStylesForNoShadowDOM} from '@/src/decorators/inject-styles-for-no-shadow-dom.js';
+import type {InitializableComponent} from '@/src/decorators/types.js';
+import {withTailwindStyles} from '@/src/decorators/with-tailwind-styles.js';
+import {
+  renderExpandableText,
+  type TruncateAfter,
+} from '../../common/expandable-text/expandable-text.js';
+import type {CommerceBindings} from '../atomic-commerce-interface/atomic-commerce-interface.js';
+import styles from './atomic-product-description.tw.css';
+import '../atomic-product-text/atomic-product-text.js';
```

Comment:
Mmm... we have the same import in the atomic-product-link that @fpbrault is migrating. This might cause runtime errors as we'll be trying to register the atomic-product-text custom element multiple times.